### PR TITLE
Test npm and pypi dependencies using Snyk

### PR DESCRIPTION
**Pull request by cadecairos**
This adds a quick scan of our npm and pypi dependencies for vulnerabilities. 

I've added documentation about how to respond to vulnerability related CI failures to README.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
+  - DATABASE_URL=postgres://postgres@localhost:5432/network
+  - USE_S3=False
+  - CORS_WHITELIST="*"
+  - XSS_PROTECTION=True
+  - CONTENT_TYPE_NO_SNIFF=True
+  - SET_HSTS=False
+  - SSL_REDIRECT=False
+  - X_FRAME_OPTIONS=DENY
+  - ALLOWED_HOSTS=localhost
+  - secure: 61FiWTuYBYgmiizqAH2MTeMJrfIcReoeDV2jz8umhMiXkvyA6LQQM5tjEtNfbHMx2SGqi5YyVlVeuTV71FCmS/V8wxVd652SNUK3UXW43rzmOEMYvOvB9rQ8IZVO5U9mIeXSnVJi8vYiZbIv4udrxka6O1PoUvsvE+R6Yl2FPQwe7HbEDF/I8VxNCbPPoj/KvUuCzGpI7p3Y6KSsDEiqMa/fJO3syOENIwaWhzvZzCXto+jYUJbtMenMadft506NYrYD+o5egpmJ8sKoxK1xzbR7iMUzgmAwGynY2dPF2M6nl44V6VutprrRfLNMCRRnamiJiZVfj/8+L/MCmS2uSgir8GSIY8DPivULwUlpvz67tYv3RgZbSVemev5I77VXK8MJ+vOAmtinI48ZZHhQ+LPHz2tiqxVhtCpYfDmSSmIqhvMJzDv0yVvEzz8c5B9Ui2ozrFB3kSrR1Gsiwl1VDsP1ymJSpWMLRgcHbMszZh6T5htgwJ9Lzz1mxZDzxTBfo2hn4SQ1Gdbu09SY/he5NYvcQEZ2jiFCE9rF4OR9UGIC9wHGrJ/BxW2mLPMwaXa9bL8+feNz52TR/GiVRy3hbgokGadVv0iZYLvEKty/RvO7KytFkusp516/qzT+2Cyw1cA6uAvwJL2vrl1PXQa8qAcN560QF6KnxIAY5mtP0rk=
```
Comment (cadecairos) [inline .travis.yml]: this is the only thing I added - our whitespace formatting was removed by the travis CLI tool when I encrypted this token.

**Comment by cadecairos** [comment]
seeing some weird behaviour with encrypted vars on appveyor, and travis is also not running tests, as far as I can see... will look into this more tomorrow.

**Comment by alanmoo** [comment]
Ping me when it's ready to be looked at

**Comment by cadecairos** [comment]
@alanmoo sorted out the appveyor configuration issues.

**Pull request merged**